3-aria/aria.md

```diff
@@ -105,30 +114,29 @@ Below are a few commonly used attributes and what they achieve. You can view a m
 ```
 
 ### [aria-hidden](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-hidden_attribute)
+
 > Adding `aria-hidden="true"` to an element removes that element and all of its children from the accessibility tree.
 
 This is especially useful for decorative content, or offscreen content such as modals or menus. It's worth noting that `aria-hidden="false"` however [can perform inconsistently across browsers](https://www.w3.org/TR/wai-aria-1.1/#aria-hidden).
 
 ---
 
-## Allowed attributes for HTML elements
+## Useful Tools
+
 W3C provide [a helpful table](https://www.w3.org/TR/html-aria/#docconformance) of HTML elements, their implicit ARIA semantics, and their allowed roles.
 
+[a11support.io](https://a11ysupport.io/) allows you to search by ARIA attribute or HTML element and see support across different assistive technologies and browsers.
```

Comment:
Was just having a quick scan of the content and noticed there's a 'y' missing from the URL here.  
Trust me to spot the typo! 🤓